AdventureCore/AdventureCore.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module version="4">
+  <component name="FacetManager">
+    <facet type="minecraft" name="Minecraft">
+      <configuration>
+        <autoDetectTypes>
+          <platformType>ADVENTURE</platformType>
+        </autoDetectTypes>
+      </configuration>
+    </facet>
+  </component>
+</module>
```

Comment:
I'm not sure if this file should be pushed to version control. It seems like it was not public before 1.4, but it is needed to identify the folder as an intelliJ project (or module in this case). I think reviews can be collected, but should ignore the changes to all files matching "*.iml". In the case that i decide to remove all "IML-FILES" from the project this change will be done asynchronously from the development cycle. So as long as this is save to merge to main, it will be merged.